modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/app/App.js

```diff
@@ -0,0 +1,1349 @@
+'use strict';
```

Comment:
We should eliminate all these `use strict`; because these are transpiled as ES modules (which are strict by default), Babel will insert this for us in the generated output.

Reply:
Ugh, yeah... I know I removed some, specially in tests, but didn't do a pass through sources, trying to touch as little as possible in this first pass... If tests fail spectacularly I'll do this, otherwise we can follow up.